pkg/provisioner/ironic/validatemanagementaccess_test.go

```diff
@@ -574,8 +574,8 @@ func TestValidateManagementAccessExistingPortButHasName(t *testing.T) {
 		t.Fatalf("could not create provisioner: %s", err)
 	}
 
-	_, _, err = prov.ValidateManagementAccess(false, false)
-	assert.EqualError(t, err, "failed to find existing host: node found by MAC but has a name: wrong-name")
+	_, err = prov.findExistingHost()
+	assert.EqualError(t, err, "MAC address 11:11:11:11:11:11 conflicts with existing node wrong-name")
 }
 
```

Comment:
It's not clear to me why here the test switched to `findExistingHost`. I would consider better, in this suite, to continue to verify `ValidateManagementAccess` behavior, also because it's part of the public interface, ie something like:
```

res, _, err := prov.ValidateManagementAccess(false, false)
assert.Nil(t, err)
assert.Equal(t, res.ErrorMessage, "MAC address 11:11:11:11:11:11 conflicts with existing node wrong-name")

```